.github/reusable-steps/gradio-action/action.yml

```diff
@@ -18,59 +18,89 @@ runs:
       run: |
         cd ${{ inputs.project }}
 
-        if [ "${{ runner.os }}" == "Linux" ]; then
-          # Start the Gradio app in the background
-          xvfb-run python ${{ inputs.script }} 2>&1 | tee gradio_log.txt &
+        if [[ "${{ inputs.script }}" == *"test.py" ]]; then
```

Comment:
drop all changes in action.yml

Reply:
Reverted back action.yml as in master with the addition of windows foreground logic